.github/workflows/github-ci.yaml

```diff
@@ -17,6 +17,39 @@ on:
   workflow_dispatch:
 
 jobs:
+  test_windows:
+    name: pytest on windows
+    runs-on: windows-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+    - name: Setup Python (3.11+)
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11  # latest stable python
```

Comment:
This comment is wrong since October: https://www.python.org/downloads/release/python-3120/

Reply:
Uh, interesting! I didn't realize 3.12 is already stable :tada:  Thank you :hugs: